README.md

```diff
@@ -45,35 +43,176 @@ and its colocated CNS Object store.  They fill the role of our micro-service pro
+- Configure `gcloud` user, zone, and project.  These are detected by the deployment script and required for setup.
 
-```
-GCP_USER=<gcp user name>
-GCP_ZONE=<gcp zone>
-GCP_REGION=<gcp region>
-```
 
-#### Step 1: Deploy Gluster-Kubernetes cluster in GCP
+`# gcloud config set account <user account>`
+
+`# gcloud config set project <project>`
+
+`# gcloud config set compute/zone <zone>`
+
+
+### Step 1: Deploy Gluster-Kubernetes cluster in Google Compute Engine (GCE)
 This step sets up the **External Service Provider** portion of our topology (see [diagram](./docs/Service-Catalog-Published.png)).
 
 To kick off deployment, run `gk-cluster-deploy/deploy/cluster/gk-up.sh`.  The script has a number of configurable
-variables relative to GCP Instance settings.  These can be defined inline with `gk-up.sh` or as environment variables.  To run the script as fire and forget, pass `-y` to skip configuration review.
+variables relative to GCE Instance settings.  These can be defined inline with `gk-up.sh` or as environment variables.  To run the script as fire and forget, pass `-y` to skip configuration review.
 
-Runtime take around 5 to 10 minutes.  Go get some coffee.
+Runtime takes around 5 to 10 minutes.  Go get some coffee.
+
+Alternatively, you can manually deploy the GCE portion by following [these instuctions](//TODO).  // TODO
 
 When deployment completes, commands to ssh into the master node and the URL of the CNS Object Broker will be output. **Note the URL and PORT.**
 
-#### Step 2: Spin up a Local Kubernetes Cluster
+### Step 2: Deploy the Service-Catalog
 This step sets up the **K8s Cluster** portion of our topology (see [diagram](./docs/Service-Catalog-Published.png)).
 
-Create a local Kubernetes cluster:
-```
-KUBE_ENABLE_CLUSTER_DNS=true kubernetes/hack/local-up-cluster.sh
-```
-
 *In a separate terminal:*
+1. Change directories to the `service-catalog` repository.
+
+2. Follow the [Service-Catalog Installation instructions](https://github.com/kubernetes-incubator/service-catalog/blob/master/docs/introduction.md#installation).  Once the Service-Catalog is deployed, return here.
+
+
+## Using Service Catalog
+
+**STOP!** If you have made it this far, it is assumed that you now have
+
+- A 4 node Kube cluster in GCE, running:
+  - Gluster-Kubernetes and its S3 components.
+  - Our CNS Object Broker.
+
+You can check the status of all these components at once by executing
+
+`# gcloud compute ssh <master node name> --command="kubectl get pod,svc --all-namespaces"`
+
+### Create the ServiceBroker (API Object)
+
+Change to the `cns-object-broker` directory created when cloning the repo.
+
+0. Retrieve the URL and PORT of the cns-object-broker.
+
+    If `gk-up.sh` was run, it will have been output at the end of the script.  
+
+    To get the url and port manually, first note the **external ip** of any GCE node in the cluster.  The broker is exposed via a *NodePort Service* and so is reachable via any node.
```

Comment:
It would be helpful if gk-up exported the s3URL and PORT as variables.

Reply:
Do you mean for debugging the storage component?